core/tasks.py

```diff
@@ -199,12 +204,22 @@ def process_invoice(invoice_id):
         return
     tv_invoice_id = invoices[0].tv_id
     view_id = invoices[0].view_id
+    is_manual = invoices[0].is_manual
     if total_amt == balance:
-        updateTvInvoiceStatus(tv_invoice_id, 'UNPAID', view_id)
```

Comment:
Can make two separate functions. One for normal invoice and other for manual. It will be clearer.

Reply:
fixed